cinema/views.py

```diff
@@ -103,6 +110,10 @@ def get_serializer_class(self):
 
         return MovieSerializer
 
+    def perform_create(self, serializer):
+        """Ensure the image field is not set during movie creation"""
+        serializer.save(image=None)
```

Comment:
The `perform_create` method in `MovieViewSet` sets the `image` field to `None` during movie creation. Ensure this behavior is intentional, as it prevents setting an image during the creation process. If you want to allow image uploads during creation, consider handling this differently.